src/NUnitFramework/nunitlite.runner/AutoRun.cs

```diff
@@ -75,7 +82,48 @@ public int Execute(string[] args)
                 Console.SetError(errWriter);
             }
 
-            var _textUI = new TextUI(outWriter, options);
+            try
+            {
+                return Execute(testAssembly, new NUnitLiteOptions(args), new ColorConsoleWriter(), Console.In);
```

Comment:
Shouldn't you be passing in the options and the outWriter that you created above rather than creating new ones? This breaks the OutFile option.